static/app/components/charts/eventsRequest.tsx

```diff
@@ -245,7 +244,6 @@ class EventsRequest extends React.PureComponent<EventsRequestProps, EventsReques
     this.setState(state => ({
       reloading: state.timeseriesData !== null,
       errored: false,
-      errorMessage: null,
```

Comment:
hmm I think you might want keep this but update to `errorMessage: undefined` since you likely still want to "reset" the state of the `errorMessage` when `fetchData` is called. This would also match the behaviour with setting `errored: false`

Reply:
Got it. Thank you :)